### DISCUSSION TITLE
Linting error with github-workflow schema when using reusable workflow

### DISCUSSION THREAD

**Discussion by tyler-jewell** [community/community#45840] [Actions]
### Select Topic Area

Bug

### Body

When I use a reusable workflow in github action, I get a linting error saying that steps and runs-on are required when they are not.

![image](https://user-images.githubusercontent.com/72859335/215616423-c2c54f21-3d13-4f38-a24e-d4dc1cdf53a2.png)

**Comment by norwd** [comment]
Hi @tyler-jewell, what linter are you using that is giving you this error? It looks like VS Code (maybe a plugin) and if so it might be that the authors of this plugin need to update their tool to include reusable workflows.

**Comment by andrewakim** [comment]
Hi all :wave: - I just wanted to drop in and let you know there is an official [Actions extension for VS Code](https://github.com/github/roadmap/issues/564) coming 🚀 🎉 ! I'll do my best to circle back and post an update when it's been released.
